Content.Server/Power/Components/UpgradePowerDrawComponent.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Content.Shared.Construction.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Power.Components;
+
+/// <summary>
+/// This is used for machines whose power draw
+/// can be decreased through machine part upgrades.
+/// </summary>
+[RegisterComponent]
+public sealed class UpgradePowerDrawComponent : Component
+{
+    /// <summary>
+    /// The base power draw of the machine.
+    /// Value is initializezd on map init from <see cref="ApcPowerReceiverComponent"/>
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float BaseLoad;
+
+    /// <summary>
+    /// The machine part that affects the power draw.
+    /// </summary>
+    [DataField("machinePartPowerDraw", customTypeSerializer: typeof(PrototypeIdSerializer<MachinePartPrototype>)), ViewVariables(VVAccess.ReadWrite)]
+    public string MachinePartPowerDraw = "Capacitor";
+
+    /// <summary>
+    /// The multiplier used for scaling the power draw.
+    /// </summary>
+    [DataField("modifier", required: true), ViewVariables(VVAccess.ReadWrite)]
+    public float Modifier = 1f;
```

Comment:
```suggestion
    [DataField("powerDrawMultiplier", required: true), ViewVariables(VVAccess.ReadWrite)]
    public float PowerDrawMultiplier = 1f;
```
So naming is consistent with UpgradeBatteryComponent.